charts/agent-connector-azure-vault/Chart.yaml

```diff
@@ -42,7 +42,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.10.2-SNAPSHOT
+version: 1.10.3
```

Comment:
I'm so sorry, I should have mentioned that we leave the SNAPSHOT in the main branch.
Could you please make the version 1.10.3.-SNAPSHOT?

Reply:
```suggestion
version: 1.10.3-SNAPSHOT
```